libraries/SPI/SPI.cpp

```diff
@@ -166,6 +166,8 @@ void ArduinoSPI::begin()
   }
   else
   {
+#if 0
```

Comment:
Since this implementation works without interrupts, you may as well fully remove the code statements.

Reply:
done, only left in the comment